src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -1,13 +1,14 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.collections.IntArrayStack;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class RhymersJUnitTest {
 
     private static final int TEST_VALUE = 4;
     private static final int STACK_CAPACITY = 12;
-    private static final int EMPTY_STACK_VALUE = -1;
+    private static final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).